models/Asset.php

```diff
@@ -168,7 +168,7 @@ public function setDataModificationDate(?int $dataModificationDate): static
 
     protected function getBlockedVars(): array
     {
-        $blockedVars = ['scheduledTasks', 'versions', 'parent', 'stream'];
+        $blockedVars = ['scheduledTasks', 'versions', 'stream'];
```

Comment:
Mhhh, why is like that the `parent` is not blocked anymore? 
The parent element should be blocked and getted by the parentId instead, right?

Reply:
resolved: ah, it's already covered by https://github.com/pimcore/pimcore/pull/16596/files#diff-464ea7ee8142d9960b6349fb040c177d818aefdd437151c3417ff72b04fced26L627